src/utilities.ts

```diff
@@ -13,7 +13,10 @@ const RESERVED_SVG_MATHML_ELEMENTS = new Set([
   'font-face-format',
   'font-face-name',
   'missing-glyph',
-]);
+] as const);
+
+type ReservedSvgMathmlElements =
+  typeof RESERVED_SVG_MATHML_ELEMENTS extends Set<infer T> ? T : never;
```

Comment:
`Set` is strictly managed, and I declared `type` accordingly.

```ts
type ReservedSvgMathmlElements =
  | 'annotation-xml'
  | 'color-profile'
  | 'font-face'
  | 'font-face-src'
  | 'font-face-uri'
  | 'font-face-format'
  | 'font-face-name'
  | 'missing-glyph';
```

Reply:
Looks good